onedal/__init__.py

```diff
@@ -64,3 +64,6 @@
 
     if _is_dpc_backend:
         __all__ += ["spmd.cluster"]
+
+if daal_check_version((2024, "P", 1)):
+    __all__ += ["logistic_regression"]
```

Comment:
logistic_regression is within the linear_model module - not sure if this is necessary